### ISSUE TITLE
Map Block: Add option to prevent map zoom if you scroll while cursor is on the map

### ISSUE THREAD

**Issue by danjjohnson** [Automattic/wp-calypso#29359]
#### Steps to reproduce
1. Add a map to a page.
2. View the page.
3. Scroll down the page using trackpad or scroll wheel
4. Try to scroll while the cursor is over the map.

#### What I expected

The page will continue to scroll.

#### What happened instead

The map zoomed in/out instead

#### Browser / OS version

Chrome Version 70.0.3538.110 (Official Build) (64-bit)
MacOS 10.14

#### Screenshot / Video

![map-scroll-zoom](https://user-images.githubusercontent.com/1990037/49886800-7c2ddb00-fe32-11e8-80c8-ef9c05ac8a94.gif)

#### Context / Source

Reported in 1618575-zen

**Comment by alisterscott** [comment]
I've seen this type of behaviour on many sites that embed interactive maps - I'm not sure there's much we can do to stop it?

**Comment by danjjohnson** [comment]
Forgot to include this earlier:
https://www.mapbox.com/mapbox-gl-js/example/disable-scroll-zoom/

**Comment by 6clicks** [comment]
Yes I am pushing this ... it's the only problem with this plugin that block me to use it. with a Mac book it's not user frendly at all..
the best thing will be to add a radio button to enable it. i's " just" one line of code 
`// disable map zoom when using scroll
map.scrollZoom.disable();`

at least can you explan me ho I can add it to my .js file and point the form used by the plugin .
